### PR TITLE
controller: rename monitor to watch

### PR DESCRIPTION
**Pull request by xiang90**
/cc @hongchaodeng @hasbro17

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -257,7 +257,11 @@ func (c *Controller) createTPR() error {
 	return k8sutil.WaitEtcdTPRReady(c.KubeCli.Core().GetRESTClient(), 3*time.Second, 30*time.Second, c.Namespace)
 }
 
-func (c *Controller) monitor(watchVersion string) (<-chan *Event, <-chan error) {
+// watch creates a go routine, and watches the etcd.cluster kind resources from
```
Comment (hongchaodeng) [inline pkg/controller/controller.go]: "etcd.cluster" -> "cluster.etcd"

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -257,7 +257,11 @@ func (c *Controller) createTPR() error {
...
+// the given watch version. It emits new events changed in the resources set
```
Comment (hongchaodeng) [inline pkg/controller/controller.go]: It emits new events on the resources through...

File: pkg/controller/controller.go [hunk 3]
```diff
@@ -257,7 +257,11 @@ func (c *Controller) createTPR() error {
...
+// through the returned event chan. Errors will be reported throug the returned
```
Comment (hongchaodeng) [inline pkg/controller/controller.go]: throug -> through

**Comment by xiang90** [comment]
@hongchaodeng PTAL.

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**